core/templates/dev/head/pages/exploration_editor/history_tab/HistoryTab.js

```diff
@@ -53,9 +53,10 @@ oppia.controller('HistoryTab', [
     var explorationSnapshots = null;
     var versionTreeParents = null;
     var nodesData = null;
+    var currentPage = 0;
+    $scope.currentPageDisplay = currentPage + 1;
```

Comment:
$scope.displayedCurrentPageNumber perhaps, since "page number" is the thing that this is.

Reply:
Done.